### ISSUE TITLE
Tracking evaluation metric

### ISSUE THREAD

**Issue by SeungHunJeon** [facebookresearch/metamotivo#25]
Thank you for sharing your impressive work!

I tried using the pre-trained model, and it performs well in the provided environment.

I have a few questions regarding the training process and evaluation tracking.

As I understand it, the evaluation metrics are used to update priorities with respect to the motion data buffer, guiding the agent in a PHC-like manner.

However, it seems that only the "emd" tracking evaluation metric is used.

Are the other metrics—such as reward-based evaluations or alternative tracking metrics (e.g., PHC)—used anywhere? If so, where are they applied in the process?

I also have a question about the training procedure:

During the entire training process (with the default parameter set to 3,000,000 steps), it seems that the reset function is only called once at the very beginning. Wouldn’t the agent end up flailing around in a fallen state? I'm curious—why isn’t the environment explicitly reset more frequently?

**Comment by MateuszGuzek** [comment]
Hi Seung, thanks for trying it out!

We use other metrics for evaluation purposes: either as reported in the paper in the full post-training evaluation, or during smaller-scale evaluations that give us signal during the training. For example, reward evaluation is of different nature than tracking, so it is useful to keep track of the model's performance from different perspectives. The tracking metrics also have different meanings, so you can select which one is the most relevant in your use case.

During training we use gymnasium.wrapper.TimeLimit of HumEnv, so each environment is reset after every 300 steps, there is no need for explicit reset in the training loop.
